scripts/aptly/aptly_importer.py

```diff
@@ -313,13 +317,14 @@ def run(self):
         for dist in self.config.suites:
             # 1. Create aptly mirrors from yaml configuration file
             self.__create_aptly_mirror(dist)
-            # 2. Be sure mirror has all source packages
-            self.__log(f'Check all source packages exist')
-            if not self.aptly.exists_all_source_packages(Aptly.ArtifactType.MIRROR,
-                                                         self.__get_mirror_name(dist)):
-                self.__remove_all_generated_mirrors()
-                self.__error(f'{self.__get_mirror_name(dist)} does not have a source package. Removing generated mirrors')
-            self.__log_ok('All source packages exist in the mirror')
+            # 2. Be sure mirror has all source packages unless the requirement is overridden.
+            if not self.config.override_sources_required:
```

Comment:
It always logs that it's verifying the sources. For this case it would be good to log that it's explicitly skipping the check based on the config.